src/Datadog.Trace/Vendors/Serilog.Sinks.File/Sinks/File/PeriodicFlushToDiskSink.cs

```diff
@@ -13,6 +27,7 @@ namespace Datadog.Trace.Vendors.Serilog.Sinks.File
     /// <summary>
     /// A sink wrapper that periodically flushes the wrapped sink to disk.
     /// </summary>
+    [Obsolete("This type will be removed from the public API in a future version; use `WriteTo.File(flushToDiskInterval:)` instead.")]
```

Comment:
Should we remove classes like this to make our package smaller or is that too micro?

Reply:
I would have said it's not worth the hassle personally, especially if we're going to be doing this vendoring more often, but I'm not passionate about it either way

Reply:
I agree that we should minimize the manual changes we make